plugin-server/src/config/config.ts

```diff
@@ -62,6 +62,7 @@ export function getDefaultConfig(): PluginsServerConfig {
         KAFKA_MAX_MESSAGE_BATCH_SIZE: isDevEnv() ? 0 : 900_000,
         KAFKA_FLUSH_FREQUENCY_MS: isTestEnv() ? 5 : 500,
         APP_METRICS_FLUSH_FREQUENCY_MS: isTestEnv() ? 5 : 20_000,
+        APP_METRICS_FLUSH_MAX_QUEUE_SIZE: isTestEnv() ? 5 : 1000,
```

Comment:
question: How did we decide on 1000 for APP_METRICS_MAX_QUEUE_SIZE? Is this a variable we will refine as we go?

Reply:
I picked at random - feel free to suggest something else

Reply:
Sounds fine as a starting point, we can refine it 👍